src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,20 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+        manufacturerService.getAll().forEach(System.out::println);
+
+        DriverService driverService = (DriverService)
+                injector.getInstance(DriverService.class);
+        driverService.getAll().forEach(System.out::println);
```

Comment:
Please use main class to properly test project by invocation of all service layer methods.
Now you're just getting all drivers without adding any data, don't think you can get anything except empty list